lib/ranked-model/ranker.rb

```diff
@@ -69,6 +70,19 @@ def update_rank! value
           update_all(ranker.column => value)
       end
 
+      def insert_after(previous_ranker)
+        if previous_ranker.instance.class != instance.class
+          raise ObjectMismatchError, 'The previous object sent does not belong to the same model as self'
+        end
+
+        preceeding_object_rank = previous_ranker.relative_rank
+
+        new_rank = preceeding_object_rank > relative_rank ? preceeding_object_rank : preceeding_object_rank + 1
+
+        instance.send "#{ranker.name}_position=", new_rank
+        instance.send "#{ranker.column}_will_change!"
```

Comment:
This is already set when the position is set (see the class method).

Reply:
Yes, it is. My bad. Will remove it from here.